### PR TITLE
Fix/option classes

### PR DESCRIPTION
**Pull request by jose-carlos-sousa**
## This PR seeks to enhance the request structure in the backend.

Previously, the request included the fields **Options** and **Classes**, which complicated the association between a class and its corresponding option. Additionally, the classes were not being inserted correctly.

Now, each option field includes both the **from** and **to** classes, simplifying the association and improving the overall functionality.

Warning the frontend needs a few changes to be compatible with this pr, those changes have been implemented in the branch feature/preview of the frontend however a PR for it might take a bit.

### CODE REVIEW

File: django/university/routes/student/schedule/StudentScheduleView.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def get(self, request, format=None):
 
         try:
             response = requests.get(get_student_schedule_url(
-                request.user.username, 
+                202208817, #request.session["username"],
```
Comment (tomaspalma) [inline django/university/routes/student/schedule/StudentScheduleView.py:18]: Could you change this back to `request.user.username`?

**Review by tomaspalma — CHANGES_REQUESTED** [review]
Nice work

File: django/university/routes/student/schedule/StudentScheduleView.py (hunk 1 above)
Comment (jose-carlos-sousa) [reply django/university/routes/student/schedule/StudentScheduleView.py]: Done

**Review by tomaspalma — APPROVED** [review]
:rocket:

**Pull request merged**